index.js

```diff
@@ -12,11 +12,12 @@ module.exports = function (glob, opts) {
   // support single character matching, matching ranges of characters, group
   // matching, etc.
   var extended = opts ? !!opts.extended : false;
-
+  
   // If we are doing extended matching, this boolean is true when we are inside
   // a group (eg {*.html,*.js}), and false otherwise.
   var inGroup = false;
-
+  // flags to pass in to RegExp constructor
+  var flags = opts ? String(opts.flags):"";
```

Comment:
Nitpick: spaces around `:` and full sentences/capitalization/puncuation in comments, please.